docs/operator-manual/ui-extension/install.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: "Quickstart"
+title: ""
+---
+
+# Rancher UI Extension
+
+This section describes how to install the Kubewarden UI as an extension of [Rancher Manager](https://github.com/rancher/rancher).
+
+:::note
+This requires a running instance of Rancher Manager `v2.7.0` or greater.
+:::
+
+## Install
+
+The Kubewarden UI is installed as a global extension, however, the Kubewarden controller will be installed through the Rancher UI as a cluster scoped resource.
+
+### Create the UI extension repository
+
+With Rancher Extensions enabled, navigate to the Extensions page from the side-navigation and select "Manage Repositories" within the action menu (the three dots) in the top-right corner of the page. Create a new Git repository with `https://github.com/kubewarden/ui.git` for the Git Repo URL input and the Git Branch `main`.
```

Comment:
Since we are prompting to install from `main`, I wonder if we could add a warning paragraph with the `:::note` or similar. At least saying that it may be broken at times and this is temporary until we have releases.

Reply:
I added a note explaining the use of `main`, let me know what you think.

Reply:
Good enough! resolving